WgWall/ClientApp/src/app/transport/transport.component.ts

```diff
@@ -15,31 +15,30 @@ export class TransportComponent {
   public faPencilAlt = faPencilAlt;
 
   //properties
-  public stationName: String;
+  public stationName: string;
   public vehicles: Vehicle[];
   private settingKey = "trasport.service.station";
   private setting: Setting;
 
   //input
   public isEditActive: Boolean = false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this 'Boolean' wrapper object with primitive type 'boolean'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1533)